bundles/org.openhab.binding.ambientweather/src/main/resources/ESH-INF/thing/channels.xml

```diff
@@ -73,11 +73,11 @@
 	</channel-type>
 
 	<channel-type id="rainHourlyRate">
-		<item-type>Number</item-type>
+		<item-type>Number:Speed</item-type>
 		<label>Rain Fall Hourly Rate</label>
 		<description>Rain fall hourly rate</description>
 		<category>Rain</category>
-		<state readOnly="true" pattern="%.2f %unit%" />
+		<state readOnly="true" pattern="%.7f %unit%" />
```

Comment:
Isn't it too much to have by default 7 digits afterthe , ?

Reply:
Good question. But this was intentional.

The way UoM works... Even though I set the QuantityType to `SmartHomeUnits.INCHES_PER_HOUR` or `SmartHomeUnits.MILLI_METRE_PER_HOUR`, the default unit (i.e. `%unit%`) shows up on the sitemap as `mph` or `km/h`.

Therefore, if the default label is defined like this, `%.2f %unit%`, it always will show 0, even when it's raining. For example, 1.05 in/h is 0.00001657 mph. Increasing the number of decimal places will give someone a clue that they need to change the label to something like `%.1f mm/h` or `%.2f in/h`.

See discussion [here](https://github.com/openhab/openhab-core/issues/644#issuecomment-511374213).